### ISSUE TITLE
The search-google-by-version feature is worse than useless

### ISSUE THREAD

**Issue by paulp** [sbt/sbt#1164]
Too many times to count I get zero results from the little box, only to find there are plenty of docs on what I'm looking for. This is way worse than not having a little box.

Go try searches for

```
ProjectRef site:http://www.scala-sbt.org/0.13.1/docs
```

No results. That's the default search. Now try

```
ProjectRef site:http://www.scala-sbt.org/release/docs
```

Listen: if someone is searching for results for some version which is NOT the release version, then it's really important that their results be customized to the version they are search for. This thing is obviously completely untested - it can't find any results, let alone a specific subset of results - give up the pretense of being able to deliver granular results and have a single unconfigurable search box which searches at `http://www.scala-sbt.org/release/docs`. 

They're programmers, they're well aware that there are multiple versions of things and that they need to make sure what they're reading applies to what they're using. THAT problem, EVERYONE has. THIS problem - finding no results for the front-line documentation search - sbt has created, and for no benefit.

**Comment by jsuereth** [comment]
Interesting, it looks like google stopped crawling the versioned documentation, but continues to crawl the 'release' docs.  Maybe they think versioned docs is a ponzi scheme?

In any case, feel free to use www.google.com for searching for docs.  It's quite good.  I'll see if I can finagle the gods of SEO to fix this, or just default the search to hit the release site.

**Comment by eed3si9n** [comment]
I'm tempted to suggested that we run our own search index, which is probably a slippery slope to more problems. Akka's website seems to provide custom Google search filtered by version:
- http://doc.akka.io/docs/akka/2.3.0/scala.html
- http://doc.akka.io/docs/akka/2.2.4/scala.html

**Comment by jsuereth** [comment]
Yeah, that works when google indexes your site.  Google isn't crawling the latest version site.   We'll have to jiggle the handle until our data is flushed down the google drain _or_ figure out an alternative mechanism.

We have constantly had issues with google's indexing our information in ways that aren't helpful for our users.  This seems the most significant....

Given how SEO works, you have to be careful how we optimise this too.

**Comment by martijnhoekstra** [comment]
Can Google get to http://www.scala-sbt.org/0.13.1/docs by crawling? I've
been clicking over the site for a bit, but I haven't found a way to end up
there.

**Comment by jsuereth** [comment]
Yeah, i think the link disappeared.  even if it could, the duplicated content with the `releases` URL actually lowers the SEO of it too, which I think was why the link disappeared in the first place.   I'll try to fix before 0.13.2 docs go out.

**Comment by paulp** [comment]
Be aware the issue isn't limited to 0.13.1 and it definitely isn't a function of what's crawlable. If you do some searches with site:http://www.scala-sbt.org/XXX/docs/ for various values of XXX you will see a ragged tapestry of incomplete results. At the moment it reports 649 results for 0.13.0, 98 for release, 3 for 0.13.1, and numbers all over the map for 0.12.x versions.

**Comment by julianpeeters** [comment]
Heh, I'm here because Google hasn't been working for me and I was wondering where the search box went :)

Google: while it's true that most of the time the results aren't burried and one result will LOOK like the current site, after clicking, the result is invariably out of date. And each result tops out at a different version number, none of them linking to the current docs.

The reference manual pdf is a last resort, but of course it's clunky and requires exact search terms.

Anyways, thanks for all the work guys, just sharing my pain hoping to encourage a fix.

**Comment by eed3si9n** [comment]
The search box was removed when we moved to Pamflet, and we've been hearing surprisingly few complaints about it. Maybe because it was never functional as this issue points out.

This case is now split into two issues:
- sbt/website#5 Search box
- sbt/website#180 Search Engine Optimization

My guess was that Google was confused due to the sitemap we were producing (or suspected that we are spam site that produces same content over and over). I've renamed the sitemap in sbt/sbt.github.com@5d3b4a8a37d79c3d1b8502cf9cff9cb4c83e8b21, also we produce one site per binary compatible series. The following Google result looks ok:

```
ProjectRef site:www.scala-sbt.org/0.13/docs
```
